### DISCUSSION TITLE
Hashtag Privacy

### DISCUSSION THREAD

**Discussion by avisaxena33** [mastodon/mastodon#27951] [General]
Is there a feature where only certain users in an instance can see toots from certain hashtags. For example, if I tooted out something with #team1 , only people associated with 'team1' could see those toots? Is there anything like that or something of that nature?

**Comment by Cassolotl** [comment]
Probably !groups is what you're after, which works a lot like that on the fediverse: *Support groups #139*

**Comment by avisaxena33** [comment]
Ah so it has not been implemented yet, that's a shame. I guess I will look into gab. Thanks!

**Comment by Cassolotl** [comment]
@avisaxena33 If it helps, there are many other federating ActivityPub sites that implement groups. I was on GNU Social years ago and they did groups like this. (Can anyone respond with other non-Mastodon ActivityPub things that have the !groups feature?)

**Comment by Gargron** [comment]
For the record `!groups` is anything but what @avisaxena33 asks for. They're completely public, as is everything on GNU social. As far as I can tell from looking at the code, Gab's groups are public too.

**Comment by avisaxena33** [comment]
@Gargron Aside from this, I want to learn more about the flexibility of federations. Is it possible for me to make it such that some federations are public such that any instance can follow it and make some federations private. In essence, let's see I have instance A, B, and C. Can I make A and C private such that A/B cannot follow C and B/C cannot follow A, and make B private such that A and C can follow B?
